generators/server/templates/_README.md

```diff
@@ -126,7 +126,7 @@ To optimize the <%= baseName %> application for production, run:
     ./gradlew -Pprod clean bootRepackage<% } %>
 
 <%_ if(!skipClient) { _%>
-This will concatenate and minify the client CSS and JavaScript files. It will also modify `index.html` so it references these new files.
+This will concatenate and minify the client CSS and JavaScript files. It will also modify [index.html](index.html) so it references these new files.
```

Comment:
Sorry I did not catch it in my review, thanks Pascal. I'm wondering  whether we should even have a link here as the file may be built by webpack so only exists in target/www folder.